pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -13,6 +13,15 @@ import (
 	"github.com/lf-edge/eve/pkg/pillar/types"
 )
 
+const (
```

Comment:
Maybe it would make sense to define these constants here, https://github.com/lf-edge/eve/blob/db4f3f37c68d1de7e7ead64f6ff700de64dd443a/pkg/pillar/types/locationconsts.go#L143
But it's up to you, not critical.

Reply:
> they are not shared, actually they don't even need to be public

Shared between which components do you mean? I always considered this file as a root for all the path-realted constants among all Pillar components.

Reply:
I mean the 'Pillar components'.